util/env/env.go

```diff
@@ -126,6 +126,13 @@ func StringFromEnv(env string, defaultValue string) string {
 	return defaultValue
 }
 
+func StringsFromEnv(env string, defaultValue []string, separator string) []string {
```

Comment:
Instead of a semicolon-delimited string, does it make sense to just accept multiple flags to construct the array? I'm on the fence. Other array flags work by accepting multiple, but it's a bit weird to send a bunch of separate glob patterns.

Just for your consideration.  :-)

Reply:
This was the problem I was struggling with. The flags themselves can be specified multiple times, but you cannot do this with an environment variable. Thus the environment variable needs a way to be split, and semicolons seemed like the best option.

Reply:
Having read that explanation, I like what you've done. Maybe just add a note about the difference in config-management-plugins.md since it mentions both config methods?